prjxray/xjson.py

```diff
@@ -0,0 +1,98 @@
+import io
```

Comment:
Maybe we should rename this to something like `prjxray/sort_complex_data.py` or something now? The name `xjson` was just me being lazy I think....